app/Models/Product.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Product extends Model
+{
+    protected $table = 'products';
+    protected $fillable = ['product_name', 'image', 'price', 'description'];
+    public $timestamp = false;
+    use HasFactory;
```

Comment:
'use' must be put at the first line of class

Reply:
I fixed it